packages/record/src/components/CopyToClipboardButton/helpers/__test__/isSupported.spec.js

```diff
@@ -0,0 +1,21 @@
+import isSupported from '../isSupported';
+
+describe('<isSupported/>', () => {
+    it('should return true when window.clipboardData and window.clipboardData.setData is defined', () => {
+        window.clipboardData = {
+            setData: jest.fn(),
+        };
+        expect(isSupported()).toBeTruthy();
+    });
+    it('should return true when document.queryCommandSupported is defined and document.queryCommandSupported("copy") return true', () => {
+        document.queryCommandSupported = jest.fn(() => true);
+        expect(isSupported()).toBeTruthy();
```

Comment:
here window.clipboardData.setData does not exists by default?

Reply:
window.clipboardData.setData is an Internet Explore API